pkg/util/log/log.go

```diff
@@ -598,14 +619,24 @@ func DebugfStackDepth(depth int, format string, params ...interface{}) {
 
 // DebugcStackDepth logs at the debug level with context and the current stack depth plus the additional given one
 func DebugcStackDepth(message string, depth int, context ...interface{}) {
-	logContext(seelog.DebugLvl, func() { Debugc(message, context...) }, Logger.debug, message, depth, context...)
+	logFormatContext(seelog.DebugLvl, func() { Debugc(message, context...) }, Logger.debug, message, nil, depth, context...)
 }
 
 // Debugc logs at the debug level with context
 func Debugc(message string, context ...interface{}) {
 	DebugcStackDepth(message, 1, context...)
 }
 
+// DebugfcStackDepth logs at the debug level with context and the current stack depth plus the additional given one
+func DebugfcStackDepth(depth int, context []interface{}, message string, params ...interface{}) {
+	logFormatContext(seelog.DebugLvl, func() { Debugfc(context, message, params...) }, Logger.debug, message, params, depth, context...)
+}
+
+// Debugfc logs at the debug level with context
+func Debugfc(context []interface{}, message string, params ...interface{}) {
```

Comment:
`Debugfc` is not a very explicit name, looking at it I doubt dev will know what it does.

What about `DebugContextf` or something similar ?

Reply:
I thought the signature is clear enough but I can rename it to DebugContextf